ci/tasks/run-int.yml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source: {repository: everlag/fog-google}
```

Comment:
Concourse doesn't support that. A core assumption is that docker images live somewhere outside concourse. You can build docker images using concourse, they need to be uploaded externally before being used internally.